src/transformers/models/gptj/modeling_gptj.py

```diff
@@ -69,7 +69,7 @@ def fixed_pos_embedding(x, seq_dim=1, seq_len=None):
 def rotate_every_two(x):
     x1 = x[:, :, :, ::2]
     x2 = x[:, :, :, 1::2]
-    x = torch.stack((-x2, x1), axis=-1)
+    x = torch.stack((-x2, x1), dim=-1)
```

Comment:
Needed for TorchScript. This should be ok since [stack can take the `dim` argument since the very beginning](https://pytorch.org/docs/0.1.12/torch.html?highlight=torch%20stack#torch.stack).